samples/ingredients/disks/local_ssd.py

```diff
@@ -0,0 +1,41 @@
+#  Copyright 2022 Google LLC
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+# This is an ingredient file. It is not meant to be run directly. Check the samples/snippets
+# folder for complete code samples that are ready to be used.
+# Disabling flake8 for the ingredients file, as it would fail F821 - undefined name check.
```

Comment:
Does f821 affect a LOT of the file or only a few files? I'd love to just exempt the occasional line rather than the whole file if possible.

Reply:
I put the `# flake8: noqa` line in every `ingredients` file. That's because in most of them, I do refer to methods that are not defined in the file scope. However, when the `ingredients` get composed into `snippets` everything is properly defined and the files in `snippets` don't have any lint errors disabled. In summary:

1. Ingredients are meant to have references to undefined methods that cause F821 when linting.
2. Snippets that are the final product of ingredients + recipes don't have those problems and don't have any linting disabled.
3. If there are any lint problems with ingredients, they will be caught in the final snippet file.

I could put a single line ` # noqa: F821` next to every undefined method in the ingredients file, but then this comment would be carried over to the final snippets file. This would be confusing for the users, as on the snippet level there's no reason to disable this lint check.

That's why I decided to just disable flake8 for all the ingredients files with the `# flake8: noqa` comment at the top.